src/Interfaces.ts

```diff
@@ -78,6 +82,7 @@ export interface IExecutableSchemaDefinition {
   allowUndefinedInResolve?: boolean;
   resolverValidationOptions?: IResolverValidationOptions;
   directiveResolvers?: IDirectiveResolvers<any, any>;
+  directiveVisitors?: { [name: string]: typeof SchemaDirectiveVisitor };
```

Comment:
I wasn't sure whether folks would want to pass `directiveVisitors` to `makeExecutableSchema`, a la `directiveResolvers`, or just call `SchemaDirectiveVisitor.visitSchemaDirectives(schema, directiveVisitors)` after creating the schema, so both styles are supported.

Reply:
I just thought of a potential use case here: if you're stitching schemas together, you might want to apply directives to one or both of the schemas before you merge them, or you might want to wait until after merging, so the directives get to operate on the combined schema. In the second case, you'd probably want to call `visitSchemaDirectives` yourself, after the merge, rather than passing `directiveVisitors` to `makeExecutableSchema`.

Reply:
@stubailo I'm open to naming the option `directives`, but the value does have to be an object, since the keys of the object specify the names of the directives.